quiche/src/lib.rs

```diff
@@ -1878,6 +1895,15 @@ impl Connection {
             stopped_stream_remote_count: 0,
         };
 
+        // Enable path MTU Discovery and start probing with the largest datagram
+        // size
+        if config.pmtud {
+            let path_pmtud = &mut conn.paths.get_mut(active_path_id)?.pmtud;
```

Comment:
maybe `get_active_mut` instead

Reply:
or perhaps we can just set this when we create the `Path` object before inserting into PathMap earlier